app/models/prediction.rb

```
@@ -87,12 +87,44 @@ def predicted_score_string
+  def margin_std_deviation
+    Math.sqrt(
+      home_margin_variability +
+      away_margin_variability
+    )
+  end
+
+  ##
+  # Calculates the standard deviation of the predicted total score based on the pace factor
+  # and the offensive and defensive efficiency volatilities of both teams' seasons.
+  ##
+  # Calculates the estimated standard deviation of the predicted total score based on the offensive and defensive efficiency volatilities of both teams' seasons and the pace factor.
```

RuboCop (1.76.1)

[convention] Line is too long. [181/140]

(Layout/LineLength)